src/app/FinalLevelExam/FinalLevelExam-add/FinalLevelExam-add.component.html

```diff
@@ -0,0 +1,64 @@
+<h2 align="center">Add a new FinalLevelExam</h2>
+
+<div class="jumbotron">
+  <h4 class="display-4">New FinalLevelExam</h4>
+  <p class="lead">Please, provide your FinalLevelExam</p>
+
+  <form id="user-form" (ngSubmit)="onSubmit()" #form="ngForm">
+    <fieldset>
+      <div id="formulaire">
+
+        <!-- name input -->
+        <div class="form-group" [class.was-validated]="name.dirty || name.touched">
+          <label class="control-label" for="name">Name*</label>
+          <input id="name" name="name" type="text" class="form-control"
+                 [(ngModel)]="finalLevelExam.name" #name="ngModel" required>
+          <div class="invalid-feedback">Name field is required</div>
+        </div>
+        <!-- topic input -->
+        <div class="form-group" [class.was-validated]="topic.dirty || topic.touched">
+          <label class="control-label" for="topicId">In the topic*:</label>
+          <select id="topicId" name="topicId" type="text" class="form-control"
+                  [(ngModel)]="finalLevelExam.topic" #topic="ngModel" required>
+            <option *ngFor="let topic of topics" [value]="topic.uri" >{{topic.name}}</option>
+          </select>
+          <span class="invalid-feedback">Topic field is required</span>
+        </div>
+
+        <!-- level input -->
+        <div class="form-group" [class.was-validated]="level.dirty || level.touched" >
+          <label class="control-label" for="levelId">Level associated with this Final Level Exam:</label>
+          <select id="levelId" name="levelId" type="text" class="form-control"
+                  [(ngModel)]="finalLevelExam.level" #level="ngModel" required>
+            <option *ngFor="let level of levels" [value]="level.uri" >{{level.name}}</option>
+          </select>
+          <span class="invalid-feedback">Level field is required</span>
+        </div>
+
+        <!-- question input -->
+        <div class="form-group" [class.was-validated]="question.dirty || question.touched">
+          <label class="control-label" for="topicId">Questions:*</label>
+          <select id="questionId" name="questionId" type="text" class="form-control"
+                  [(ngModel)]="finalLevelExam.question" #question="ngModel" required multiple>
+            <option *ngFor="let question of questions" [value]="question.uri" >{{question.statement}}</option>
```

Comment:
The backend expects the questions array to contain the URIs of the selected Questions. This can be achieved using:
`[value]="question._links.self.href"`